crates/runestick/tests/test_native_types.rs

```diff
@@ -0,0 +1,116 @@
+/*!
```

Comment:
I'd suggest putting this and the other test case as a module under [`test_all` tests module in the `rune` crate](https://github.com/rune-rs/rune/tree/main/crates/rune/tests/test_all) instead. There's a couple of helpers you can use there, like the `rune!` macro to run rune code.

Note: also needs to be added to <https://github.com/rune-rs/rune/blob/main/crates/rune/tests/test_all/mod.rs> to run if there are new modules.

Reply:
Will do! :-)

Reply:
Done!